hawkular-wildfly-agent/pom.xml

```diff
@@ -113,6 +113,7 @@
     <dependency>
       <groupId>org.jboss.logging</groupId>
       <artifactId>jboss-logging</artifactId>
+      <version>3.1.4.GA-redhat-2</version>
```

Comment:
The thing is that EAP6.4 comes with that version (3.1.4.GA-redhat-2) and I need to compile the code with that version because of debugf, logf, warnf, etcf signature.

On `3.1.4.GA-redhat-2` we have these signatures:
```java
  void debugf(String format, Object... params);
  void debugf(String format, Object param1);
```

But on `3.3.0` we have these signatures:
```java
void debugf(String format, Object... params);
void debugf(String format, Object param1);
void debugf(String format, int arg);
void debugf(String format, long arg);
```

among others.

If we don't specify a version, it will compile with 3.3.0.
In the case of [this](https://github.com/josejulio/hawkular-agent/blob/83514a58f030928d9019495a1a73b06255aca3f4/hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/protocol/EndpointService.java#L86-L86) debug message it will expect the method `void debugf(String format, int arg);` (If we don't specify a version).

When running on EAP6.4 we get the following when starting:

```
22:15:26,698 ERROR [stderr] (Hawkular WildFly Agent Full Discovery Scan-platform-1) Exception in thread "Hawkular WildFly Agent Full Discovery Scan-platform-1" java.lang.NoSuchMethodError: org.hawkular.agent.monitor.log.MsgLogger.debugf(Ljava/lang/String;I)V
22:15:26,699 ERROR [stderr] (Hawkular WildFly Agent Full Discovery Scan-platform-1) 	at org.hawkular.agent.monitor.protocol.EndpointService$InventoryListenerSupport.fireResourcesAdded(EndpointService.java:86)
22:15:26,699 ERROR [stderr] (Hawkular WildFly Agent Full Discovery Scan-platform-1) 	at org.hawkular.agent.monitor.protocol.EndpointService$DiscoveryResults.discoveryFinished(EndpointService.java:151)
22:15:26,700 ERROR [stderr] (Hawkular WildFly Agent Full Discovery Scan-platform-1) 	at org.hawkular.agent.monitor.protocol.EndpointService$1.run(EndpointService.java:286)
22:15:26,700 ERROR [stderr] (Hawkular WildFly Agent Full Discovery Scan-platform-1) 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
22:15:26,700 ERROR [stderr] (Hawkular WildFly Agent Full Discovery Scan-platform-1) 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
22:15:26,700 ERROR [stderr] (Hawkular WildFly Agent Full Discovery Scan-platform-1) 	at java.lang.Thread.run(Thread.java:745)
22:15:26,700 ERROR [stderr] (Hawkular WildFly Agent Full Discovery Scan-platform-1) 	at org.jboss.threads.JBossThread.run(JBossThread.java:122)
```

In the other hand, if we compile with `3.1.4.GA-redhat-2` (or maybe even a lower version). It will attempt to use void `debugf(String format, Object param1);` which exists on both versions, thus working on both EAP6 and EAP7.

WRT to the -redhat version. I'm going to try to use a lower non-redhat version.